### ISSUE TITLE
Dynamic Prompt tab is not appearing, ERROR

### ISSUE THREAD

**Issue by juliux32** [adieyal/sd-dynamic-prompts#763]
Hello,

I am using Automatic1111 on Runpod cloud system and after installing Dynamic Prompts, it is not appearing.

This is the error that I am getting
![2024-04-15 08-00-30](https://github.com/adieyal/sd-dynamic-prompts/assets/21074960/894c2c08-1c8f-4d0a-8439-42a46f17f10f)

**Comment by wideload1971** [comment]
I don't have Runpod so this may not be your solution but I recently had this issue and my solution was to revert to an earlier build of Forge. So it may be the same for A1111.
Uninstall DymanicPrompts. Open a cmd window in your root A1111 folder. Type either
git revert <commit_hash>
or
git reset --hard <commit_hash>
With the commit hash being f11456e and I don't think you need the arrow brackets.
Install DynamicPrompts, and make sure it works. Back in the root folder cmd window, type
git pull
This should update you to the newest version with the extension working. I recommend making a backup before you do any of this, just in case. If it doesn't work, googly for 'A1111 install an older version'.

**Comment by akx** [comment]
@wideload1971 That won't help here.

@juliux32 For whichever reason, the `dynamicprompts` library used by the extension hasn't been installed. It should get automatically installed when you install the extension – how did you do it in the first place?

**Comment by juliux32** [comment]
@akx Like every other "main" extension, installing through the available extension tab. Also tried direct installation with a link, but it is the same thing

**Comment by akx** [comment]
Well, the webui should run our `install.py` when you install an extension.

If you try to reinstall it, you should presumably see an error (e.g. `Error running install.py for extension ...`) in the logs; that (and the lines before it) might help us figure out why the dependency didn't get installed.

**Comment by juliux32** [comment]
@akx I fixed it and you were right, it was not installed all the way.

1. I installed Dynamic Prompts using the extension tab
2. Went to the Dynamic Prompts extension folder 
3. Ran a command `python install.py` (to install it again? I guess)
4. Restarted Runpod

After this, I can see Dynamic Prompts on my webui. Also, a wildcards folder appeared in Dynamic Prompts folder, which was not there when installed only using extension tab.
